src/AnimatedLineGraph.tsx

```diff
@@ -227,6 +217,13 @@ export function AnimatedLineGraph({
     ],
     [pathEnd]
   )
+  const dotProps: SelectionDotProps = {
+    isActive,
+    color,
+    lineThickness,
+    circleX,
+    circleY,
+  }
```

Comment:
does it make sense to create them here and spread them afterwards again? Why not just pass directly to the View?

Reply:
fixed in https://github.com/margelo/react-native-graph/pull/35/commits/b61540f3b0cbc26594c094feba930dac090e3b92